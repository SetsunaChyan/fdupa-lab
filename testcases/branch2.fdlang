function main(){
x = input();

if (x < 128) {
    y = 1;
} else {
    y = 2;
}

if (y == 1) {
    check_interval(x, 0, 127);
    x = 1;
} else {
    check_interval(x, 128, 255);
    x = 2;
}

check_interval(y, 1, 2);
check_interval(x, 1, 2);
}