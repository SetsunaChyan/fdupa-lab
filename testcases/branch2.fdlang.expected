Line 11: YES; x in [0, 127]
Line 14: YES; x in [128, 255]
Line 18: YES; y in [1, 2]
Line 19: YES; x in [1, 2]
